azure-pipelines.yml

```diff
@@ -63,23 +63,33 @@ stages:
               Debug:
                 _BuildConfig: Debug
         steps:
-        - script: eng\common\cibuild.cmd
-            -configuration $(_BuildConfig)
-            -prepareMachine
-            $(_InternalBuildArgs)
-          name: Build
-          displayName: Build
-          condition: succeeded()
+        - ${{ if eq(variables._RunAsPublic, False) }}:
```

Comment:
I strongly prefer not having duplicate-except-for-running-tests scripts.  Why not just change the value of a variable like $(_InternalBuildArgs) conditionally?

Reply:
Good point, will do

Reply:
I clumped the `Build` and then `Build and run tests`+`Publish tests` together and used the override for the Test MSBuild param. I tested both the internal and public runs to verify everything